src/lib/structures/Josh.ts

```diff
@@ -595,36 +623,43 @@ export class Josh<StoredValue = unknown> {
    * await josh.set('key', 'value');
    *
    * await josh.filter((value) => value === 'value'); // { key: { path: 'value' } }
-   * await josh.filter((value) => value === 'value', undefined, Bulk.TwoDimensionalArray); // [['key', 'value']]
+   * await josh.filter((value) => value === 'value', Bulk.TwoDimensionalArray); // [['key', 'value']]
    * ```
    */
   public async filter<BulkType extends keyof ReturnBulk<StoredValue>>(
     hook: Payload.Hook<StoredValue>,
-    _value?: undefined,
     returnBulkType?: BulkType
   ): Promise<ReturnBulk<StoredValue>[BulkType]>;
 
   public async filter<BulkType extends keyof ReturnBulk<StoredValue>>(
     pathOrHook: Path | Payload.Hook<StoredValue>,
-    value?: Primitive,
+    valueOrReturnBulkType?: Primitive | BulkType,
```

Comment:
Changes made for this method should be reverted.

Reply:
Removed